cluster/mesos/docker/docker-compose.yml

```diff
@@ -157,10 +157,12 @@ scheduler:
   volumes:
   - ./static-pod.json:/opt/static-pods/static-pod.json
 keygen:
-  image: mesosphere/kubernetes-mesos-keygen
+  image: mesosphere/kubernetes-keygen:v1.0.0
```

Comment:
There are three different keygen calls: 
1. service-account key
2. root certificate authority
3. apiserver certificate

The first two can be done beforehand, but the apiserver cert requires knowledge of the IP that the API server is running on. In this case we pass the hostname `apiserver` into the keygen script and it does a `resolveip apiserver` to get the IP. Using docker's linking, dependency ordering, and shared volumes was the easiest solution.

I'm open to suggestions if you can think of an easier way.